frontend/src/lib/components/CommandBar/types.ts

```diff
@@ -0,0 +1,16 @@
+export enum BarStatus {
+    HIDDEN = 'hidden',
+    SHOW_SEARCH = 'show_search',
+    SHOW_ACTIONS = 'show_actions',
+}
+
+export type ResultType = 'action' | 'cohort' | 'insight' | 'dashboard' | 'experiment' | 'feature_flag'
```

Comment:
Is there a reason not to include other resources like Notebooks?

Reply:
Yes and no. Currently I'm searching the `name` field of the entity, but in notebooks it's instead called `title`. I didn't want to blow up this PR and rather test out the search mechanism in production, since trigram aka fuzzy search might be better anyways. Once we know how the search should work, I'll add all other entities.